winpr/libwinpr/utils/wlog/Layout.c

```diff
@@ -38,6 +38,12 @@
 #include <sys/syscall.h>
 #endif
 
+#ifndef MIN
```

Comment:
Remark that is probably out of the scope of this PR: `MAX` is defined in the FreeRDP part (not winPR unfortunately). But I noticed that we have various occurrence of `MIN` definitions in winPR.

Reply:
yes, that is a todo.